.github/workflows/inspektor-gadget.yml

```diff
@@ -216,7 +216,7 @@ jobs:
         path: /home/runner/work/inspektor-gadget/inspektor-gadget/${{ matrix.local-gadget-target }}.tar.gz
 
   build-docker-image:
```

Comment:
nit: even if not related to this commit. `build-docker-image` is not precise as it's a container image and not a docker image.

Reply:
I agreed. I will take advantage of this PR to change it.